addons/mail/static/src/core/common/discuss.js

```diff
@@ -59,6 +59,7 @@ export class Discuss extends Component {
         });
         this.notification = useService("notification");
         this.threadActions = useThreadActions();
+        this.env?.config?.setDisplayName(_t("Discuss"));
```

Comment:
What does it do?
Is it the title of the page? The name of the breadcrumb item?
I'm wondering if we wouldn't want to also add the current channel name, rather than just Discuss?